backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -221,7 +222,20 @@ trait StandardAsyncExecutionActor
     */
   lazy val commandDirectory: Path = jobPaths.callExecutionRoot
 
-  lazy val memoryRetryFactor: Option[GreaterEqualRefined] = None
+  lazy val memoryRetryErrorKeys: Option[List[String]] = configurationDescriptor.globalConfig.as[Option[List[String]]]("system.memory-retry-error-keys")
+
+  lazy val memoryRetryFactor: Option[MemoryRetryMultiplierRefined] = {
+    jobDescriptor.workflowDescriptor.getWorkflowOption(WorkflowOptions.MemoryRetryMultiplier) flatMap { value: String =>
+      refineV[MemoryRetryMultiplier](value.toDouble) match {
```

Comment:
`String.toDouble` seems like it's probably an unsafe function to me. Does the `refineV` catch the exception if the value is not a valid `Double`?

Reply:
That's a great point. Sorry I missed that. I am adding a `Try` around it and a test case with String & Boolean inputs as you suggested elsewhere